### ISSUE TITLE
Login does not work on current Firefox Nightly (81.0a1 (2020-07-29) (64-bit))

### ISSUE THREAD

**Issue by fluffynuts** [deckgo/deckdeckgo#826]
### Affected part of DeckDeckGo
- [x] Login
- [ ] Studio
- [ ] Remote control
- [ ] Documentation
- [ ] Demo
- [ ] Starter kit
- [ ] Poll
- [ ] Web Components
- [ ] Templates

### Reproduction

Steps to reproduce:
1. Open deckdeckgo.com with Firefox Nightly
2. Attempt to log in (I tried via GitHub)
3. Log in at auth provider
4. When redirected back to deckdeckgo.com, I'm still presented with the dialog to log in
 
### Expected Behavior
To log in and start making sweet, sweet slides!

### Actual Behavior
No login, no slides, sad trombone sounds

### Environment

- Firefox Nightly (81.0a1 (2020-07-29) (64-bit))
- Windows 10 version 2004

**Comment by peterpeterparker** [comment]
Nice thank you! We should keep an eye on this, let's see if it is a nightly "issue" or if it needs an update of FirebaseUI

**Comment by fluffynuts** [comment]
I agree that sometimes FFN introduces oddnesses that go away after a round of reports & bug-fixing (which reminds me: if you can get more info, it would be good to log an issue with Mozilla -- I'll gladly help, though I have no idea where to start with your stack); sometimes, however, FFN acts as the canary in the mine, warning of the impending doom that's going to eventually come everywhere (eg ssl vs cookies, as I'm finding out in our own products)
